torch/fx/experimental/symbolic_shapes.py

```diff
@@ -236,19 +272,25 @@ def eval(cls, base, divisor):
     'neg'
 }
 
-float_magic_methods = {"add", "sub", "mul", "truediv", "ceil", "floor", "eq", "gt", "lt", "le", "ge", "pow"}
+float_magic_methods = {"neg", "add", "sub", "mul", "truediv", "ceil", "floor", "eq", "gt", "lt", "le", "ge", "pow"}
+
+always_float_magic_methods = {'truediv'}
+always_int_magic_methods = {'floordiv', 'ceil', 'floor'}
+
+magic_methods_on_builtins = {"min", "max"}
```

Comment:
should we just kill these, since they desugar into < and >? Or we can always save it for a later PR.

Reply:
This add spurious attributes indeed. But at least they get tested because they're in this list. So even if we stop adding attributes for these below, we should keep them in this list for testing to run.